WordPress/src/main/res/values/dimens.xml

```diff
@@ -507,7 +507,7 @@
     <dimen name="two_line_list_item_height">72dp</dimen>
     <dimen name="page_list_divider_height">40dp</dimen>
     <dimen name="page_list_row_min_height">80dp</dimen>
-    <dimen name="page_list_featured_image_size">40dp</dimen>
+    <dimen name="page_list_featured_image_size">48dp</dimen>
```

Comment:
48dp looks kind of smallish, but correct. For reference, the post list is using 64dp as the featured image size. FYI @osullivanchris

Reply:
Yup I confirmed this with @osullivanchris 👍🏼